design/src/server/pokemon_service.md

```diff
@@ -1,4 +1,4 @@
-## Code generating the Pokémon Service
+# Code Generating the Pokémon Service
```

Comment:
Nitpick: In `design/src/server/code_generation.md` we use `Generating xyz Code`, but here we use `Code Generating the xyz`.